src/client/src/containers/AzureLogin/index.tsx

```diff
@@ -42,11 +46,38 @@ class AzureLogin extends React.Component<Props> {
       track: true
     });
   };
+  signOutClick = () => {
+    this.props.vscode.postMessage({
+      module: EXTENSION_MODULES.AZURE,
+      command: EXTENSION_COMMANDS.AZURE_LOGOUT,
+      track: true
+    });
+  };
+  keyDownClick = (event: any) => {
+    if (event.keyCode === 13 || event.keyCode === 32) {
```

Comment:
Optional nit: Sometimes it's useful to leave a comment about what the keycodes represent so devs don't have to look them up.

Reply:
Feel like these are going to be replaced by .key events so i think its okay

Reply:
Aside from my file changes, there's a lot of areas in the codebase where the format is similar so I think instead of adding comments for just one of them, we can replace all event.keyCode with eventkey (like sahil said) and check for the type of key pressed as a future improvement in a different pr

Reply:
@SahilTara recommended this before so we have an issue for this #817

Reply:
Good to know! Could help a future dev change the code a bit more easily? 🙃

Reply:
Honestly, after some thought, I don't think its a good idea to do it the wrong way just because its being done like that everywhere else. I think that if you keep it as keyCode here, it will just add to code debt. Why do it right later when you can do it properly right now?

Reply:
@SahilTara I'm thinking of doing it in a separate PR just for the issue because currently there are about 100 files that would need to change based on that. It would make this PR a lot more difficult to review

Reply:
No no I'm not saying change it for everywhere in the code base (that would be an unreasonable request), i'm saying for the new code that you're adding.

Reply:
Ohhh I see! haha okay I'll make the change then